v1.1/pvp-specification.md

```diff
@@ -176,7 +176,9 @@ version `1.0.2014` with tags `01` and `27`.
 Decision tree
 -------------
 
-The (incomplete!) decision tree summarises the PVP rules in a concise form
+The (incomplete!) decision tree summarises the PVP rules in a concise form.
+It is not a precise representation of the spec and may contain informal
+conventions and recommendations.
```

Comment:
This is the addition in this PR and I don't see it adding any value. The prior line already calls out "incomplete" and "summarises", both of which point to potential vagueness and infelicity.

I'd rather see this specifically call out how the decision tree differs from the formal spec.

Reply:
IMHO "incomplete" and "summarises" make an impression that the actual PVP demands more, but certainly not less, which is wrong. My experience is that people often skip the specification and refer to the tree as a source of truth, so being a bit louder in the disclaimer might help.

I absolutely agree that this area needs more future work. I'd suggest not to scope creep now though.